src/uiTest/resources/project.rhino.js

```diff
@@ -298,16 +303,26 @@ function Project(underlyingProject) {
     let fullName = 'Git Machete: ' + name;
     const getButton = function () {
       // findAll() returns a LinkedHashSet
-      const result = robot.finder().findAll(component =>
-        component instanceof ActionButton && fullName.equals(component.getAction().getTemplatePresentation().getText())
-      ).toArray();
+      const result = robot.finder().findAll(component => {
```

Comment:
I don't like putting a side-effecting actions (`update*`) into `findAll`'s predicate... would it be easily doable to use sth like `foreach` instead on the components that need to be updated? 🤔